docs/troubleshooting.md

```diff
@@ -3,3 +3,19 @@
 ## Deleting an Empire CloudFormation stack
 
 If you've created an Empire CloudFormation stack and deployed an app to it, you have created an ECS Service with an attached ELB inside the VPC of your Empire stack. Before you can delete the stack, you must no longer have any services or ELBs running inside of it. You can do this by running `emp destroy <app>` for each app in your Empire cluster.
+
+## Launching Empire
```

Comment:
This is pretty specific to running Empire with the empire_ami and stacker. Can you change the sub-title here to `Launch Empire with stacker and empire_ami?`.